ui/src/app/pages/analysis/notebook-resource-card.tsx

```diff
@@ -194,7 +194,7 @@ export const NotebookResourceCard = fp.flow(
               fromAccessTierShortName={resource.accessTierShortName}
               resourceType={getType(resource)}
               onClose={() => this.setState({ showCopyNotebookModal: false })}
-              onCopy={() => this.props.onUpdate()}
+              onCopy={this.props.onUpdate}
```

Comment:
Please revert - I am uncertain that this is a safe change to make, because it's difficult to handle `this` functions correctly

Reply:
This is creating issue, while showing the 'do you want to go to copied workspace' modal on copy to another workspace action.